mlibrary/my_account
# April 2023 dependency updates.

# Overview
Monthly dependency updates to help maintain Account.

## NPM
`eslint`, `sass`, and `webpack` have been updated to their latest versions (as of April 3, 2023).

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Install the updated packages.
- Run the site and make css and javascript changes.
  - Build the CSS and JS.
  - Make sure everything works.
